easystacks/software.eessi.io/2023.06/grace/eessi-2023.06-eb-4.9.4-2023a.yml

```diff
@@ -27,3 +27,33 @@ easyconfigs:
   - ALL-0.9.2-foss-2023a.eb
   - CDO-2.2.2-gompi-2023a.eb
   - BWA-0.7.17-20220923-GCCcore-12.3.0.eb
+  - METIS-5.1.0-GCCcore-12.3.0.eb
+  - SCOTCH-7.0.3-gompi-2023a.eb
+  - CGAL-5.6-GCCcore-12.3.0.eb
+  - ParaView-5.11.2-foss-2023a.eb
+  - gnuplot-5.4.8-GCCcore-12.3.0.eb
+  - Boost.MPI-1.82.0-gompi-2023a.eb: 
+      options:
+        # source URLs for Boost.MPI have changed, corresponding PR is
+        # https://github.com/easybuilders/easybuild-easyconfigs/pull/22240
+        # Boost.MPI is a dependency of ESPResSo
+        from-commit: e610fe1ac5393d1de668a466fdaaea74c580ee03
+  - ESPResSo-4.2.1-foss-2023a.eb
+  - Rivet-3.1.9-gompi-2023a-HepMC3-3.2.6.eb
+  - Pillow-10.0.0-GCCcore-12.3.0.eb
+  - sympy-1.12-gfbf-2023a.eb
+  - networkx-3.1-gfbf-2023a.eb
+  - expecttest-0.1.5-GCCcore-12.3.0.eb
+  - PyYAML-6.0-GCCcore-12.3.0.eb
+  - pytest-flakefinder-1.1.0-GCCcore-12.3.0.eb
+  - pytest-rerunfailures-12.0-GCCcore-12.3.0.eb
+  - pytest-shard-0.1.2-GCCcore-12.3.0.eb
+  - scikit-learn-1.3.1-gfbf-2023a.eb
+  - MUMPS-5.6.1-foss-2023a-metis.eb:
+      options:
+        # source URLs for MUMPS have changed, corresponding PR is
+        # https://github.com/easybuilders/easybuild-easyconfigs/pull/22582
+        # MUMPS is a dependency of snakemake
+        from-commit: 0437ff1ad34283398f55d4a6e01e6540b1ae9688
+  - snakemake-8.4.2-foss-2023a.eb
+  - LAMMPS-2Aug2023_update2-foss-2023a-kokkos.eb
```

Comment:
Should we have used more recent commits for LAMMPS (easyconfig and easyblock) as in https://github.com/EESSI/software-layer/pull/906/files?

Reply:
Apparently rebuild in #906 was only necessary for sapphire rapids. So it is not needed to rebuild for Grace.

Reply:
Thanks @bedroge !

I think, we could run some tests with the current installation. As long as it doesn’t crash, we could keep it and add a new version that includes support for Grace.

Reply:
Of course, we can check if A64FX as target would build, and if so, how it performs compared to the current installation.